### PR TITLE
Add NEWS for v0.8.3

### PR DESCRIPTION
**Pull request by blegat**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
 MathOptInterface (MOI) release notes
 ====================================
 
+v0.8.3 (March 6, 2019)
+----------------------
+
+- Use zero constant in scalar constraint function of `MOI.Test.copytest` (#691).
+- Fix variable deletion with `SingleVariable` objective function (#690).
+- Fix LazyBridgeOptimizer with bridge that add no constraints (#689).
```
Comment (mlubin) [inline NEWS.md]: s/bridge/bridges/

File: NEWS.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+- Error messages improvements (#673, #685, #686, #688).
```
Comment (mlubin) [inline NEWS.md]: s/message/message/

File: NEWS.md [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+- Documentation improvements (#682, #683, #687).
+- Basis status:
+  * Remove `VariableBasisStatus` (#679).
+  * Test `ConstraintBasisStatus` and implement it in bridges (#678).
+- Fix inferrence of `NumberOfVariables` and `NumberOfConstraints` (#677).
```
Comment (mlubin) [inline NEWS.md]: s/inferrence/inference/

File: NEWS.md [hunk 4]
```diff
@@ -1,6 +1,20 @@
 MathOptInterface (MOI) release notes
 ====================================
 
+v0.8.3 (March 6, 2019)
+----------------------
+
+- Use zero constant in scalar constraint function of `MOI.Test.copytest` (#691).
+- Fix variable deletion with `SingleVariable` objective function (#690).
+- Fix LazyBridgeOptimizer with bridges that add no constraints (#689).
```
Comment (Peiffap) [inline NEWS.md]:
```suggestion
- Fix `LazyBridgeOptimizer` with bridges that add no constraints (#689).
```

**Comment by blegat** [comment]
> Don't forget about the issue with [ci skip] right before a tag.

Indeed, good point

**Comment by blegat** [comment]
merged manually

**Pull request closed without merging**